crates/sui-core/src/transaction_outputs.rs

```diff
@@ -14,6 +15,7 @@ pub struct TransactionOutputs {
     pub transaction: Arc<VerifiedTransaction>,
     pub effects: TransactionEffects,
     pub events: TransactionEvents,
+    pub accumulator_events: Vec<AccumulatorEvent>,
```

Comment:
Isn't `accumulator_events` already in `effects`?

Reply:
yes, this is just an optimization. see https://github.com/MystenLabs/sui/pull/21701/files#diff-b33b46215a1ae7d834655dac299d7b141a20ddd89e5440f77050d46e431267ceR137-R141